tracking_2d:
  ros__parameters:
    frame_rate: 30
    track_buffer: 30
    track_thresh: 0.5
    high_thresh: 0.6
    match_thresh: 0.8
    output_frame: "map"
